docker-compose/_posts/2017-07-05-building-a-compose-file-for-runnable.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Building a Compose File for Runnable
+short: Building a Compose File for Runnable
+category: docker-compose
+order: 4
+---
+
+Runnable can deploy full-stack environments for branches of any repository that contains a Dockerfile. Let’s go over some practices you should follow in your Compose files to make this work for your team.
+
+---
+
+## Compose File Organization Patterns
+
+There are two patterns that teams tend to follow for organizing their Compose files:
+
+1. A centralized approach, with a single Compose file in a central repository that defines the whole stack.
+2. A decentralized approach, in which each service’s repository has its own Compose file that defines the environment it needs.
+
+Runnable works with either pattern because of how it makes use of the build feature in Compose.
```

Comment:
Yeah I think you're right. The only part I relly care about is that their Compose file can be anywhere, but I think that comes across in the next section.